### PR TITLE
Added file of Marcello

### PR DESCRIPTION
**Pull request by BladeBaT**
test

### CODE REVIEW

**Review by pgangarella — APPROVED** [review]
go on

**Review by demetrio-marra — APPROVED** [review]

**Review by facundorossetti — APPROVED** [review]
LGTM!

**Comment by facundorossetti** [comment]
- 

- [ ] Work done.

File: MarcelloSpanu.cs [hunk 1]
```diff
@@ -0,0 +1 @@
+Console.WriteLine("ciao");
```
Comment (facundorossetti) [inline MarcelloSpanu.cs:1]: Well done!

**Review by TeobaldoIlGiovane — APPROVED** [review]

File: MarcelloSpanu.cs (hunk 1 above)
Comment (demetrio-marra) [inline MarcelloSpanu.cs:1]: why not 

private readonly string welcomeMessage = "ciao";

public static void Main(string[] args) {

    Console.WriteLine(welcomeMessage);
}

File: MarcelloSpanu.cs [hunk 2]
```diff
@@ -1 +0,0 @@
-Console.WriteLine("ciao");
```
Comment (facundorossetti) [inline MarcelloSpanu.cs]: I just removed the console because it was unnecessary.

**Review by facundorossetti — APPROVED** [review]

**Comment by facundorossetti** [comment]
Now you can merge the PR, well done!

**Pull request merged**